csrankings.csv

```diff
@@ -12995,3 +12995,36 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Özgür Simsek,University of Bath,https://www.mpib-berlin.mpg.de/en/staff/oezguer-simsek,z1BYZG0AAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap/,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,http://bmi.osu.edu/umit/,OLDMURQAAAAJ
+Mark Sanderson,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/sanderson-professor-mark,HmqvvQkAAAAJ
+Lawrence Cavedon,RMIT University,http://www1.rmit.edu.au/staff/lawrence-cavedon,ZAEiiEUAAAAJ
+J. Shane Culpepper,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/c/culpepper-dr-shane,OFlBSEEAAAAJ
+Falk Scholer ,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/s/scholer-associate-professor-falk,qOLjlqsAAAAJ
+James A. Thom,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/t/thom-associate-professor-james,7ZXutEAAAAJ
+Xiuzhen Zhang,RMIT University,http://www1.rmit.edu.au/staff/xiuzhen-zhang,rIuehYoAAAAJ
+Zhifeng Bao,RMIT University,http://www1.rmit.edu.au/staff/zhifeng-bao,ck-yNM0AAAAJ
+Andy Song,RMIT University,http://www1.rmit.edu.au/staff/andy-song,sZ7dN0sAAAAJ
+Flora Dilys Salim,RMIT University,http://www1.rmit.edu.au/staff/flora-salim,Yz35RSYAAAAJ
+Isaac Balbin,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/b/balbin-associate-professor-isaac,YCMaqMYAAAAJ
+Ke Deng,RMIT University,http://www1.rmit.edu.au/staff/ke-deng,MuWCgskAAAAJ
```

Comment:
It appears all the links starting with www1 are broken.

Reply:
Fixed, can you please check that again? Thank you.